README.adoc

```diff
@@ -1,129 +1,104 @@
+:toc: 1
 
-* Name file by slugifying title
+== Contributing to AeroGear Documentation
 
-* Prefix filenames with con_
+* Determine which functionality is the contribution related to:
+** Is it related to a certain service only?
+** Is it common to all mobile services?
+** Is it common to all services or SDKs but with some information that is specific to each?
 
-Procedure::
-The important information that gets me using AeroGear software, for example, write your procedure by answering the following questions:
 +
-. Who should perform this procedure?
-. Why am I performing this procedure? benefits or context in a higher level user story
-. What is the object of the task? eg adding a dashboard.. to what?
-. What do I need to know and/or do before installing or provisioning the software?
-. What do I need to do to provision the software?
-. What are the minimum configuration tasks that I need to do to get value from the software?
-. What additional information would be useful after completing procedure (eg, next steps)
-
-* Name file by slugifying title
+With this information, you will be able to determine where the file should located in the repo, for example `modules/ROOT/push`.
++
+//If you intend to reuse a part of a file, create a partial in modules/ROOT/_partials and refer to it using an `include` directive.
 
-* no prefix on filenames
+* If you want to make a file name change (for example adding or renaming a file), update `modules/ROOT/nav.adoc file` to include it in the left hand menu on link:docs.aerogear.org[^]. 
 
-See link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-modules[Creating Procedures]
+- Stick to present tense, for example `When you add the dashboard, it appears on ...`, not `When you add the dashboard, it will appear on ...`
 
-Reference::
-Information I might need to debug or extend the normal usage of AeroGear software. For example:
-+
-* API syntax and definitions
-* Configuration optionss 
+- Test your changes xref:#using-antora[using Antora].
 
-* Name file by slugifying title
+- This documentation set is structured by using `#include` AsciiDoc directives:
 
-* Prefix filenames with ref_
+You might find that the content you want to change is not in the `.adoc` file with the name of the guide, but that it's "included" using the AsciiDoc `#include` directive. Editing of that included file, can have unintended effects (It may be included in several documents, and the change may not fit into context of all of them.). For example, when editing the `registering-an-app.adoc`, avoid this by: 
 
+. Regex search for `include.*registering-an-app` across the whole repo.
+. Review the list of files from this search to understand of the context of the content and make sure the content is appropriate for each rendering of that content. 
+ 
+[[mod-docs]]
```

Comment:
as a subsection of "contributing to docs"
do we want to remove this and link rh moddocs repo instead?